src/nald/formatting.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+/**
+ * Formats a NGR reference to string format
+ * @param {String} sheet - the sheet string, 2 chars
+ * @param {String} east - the eastings
+ * @param {String} north - the northings
+ * @return {String} - grid reg, eg SP 123 456
+ */
+function formatNGRPointStr (sheet, east, north) {
+  if (!sheet) {
+    return null;
+  }
+  return `${sheet} ${east.substr(0, 3)} ${north.substr(0, 3)}`;
+}
+
+/**
+ * Converts a point in NALD data to our format with {ngr1, ngr2, ngr3, ngr4, name}
+ * @param {Object} NALD point data
+ * @return {Object} point data
+ */
+function formatAbstractionPoint (point) {
+  return {
+    ngr1: formatNGRPointStr(point.NGR1_SHEET, point.NGR1_EAST, point.NGR1_NORTH),
```

Comment:
Should all NALD variables in this file be mapped through something like this to get rid of the null strings:
```
const getValue = value => value === 'null' ? null : value;
```